Assets/Tests/InputSystem/WindowsTests.cs

```diff
@@ -0,0 +1,54 @@
+#if UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN
+
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.TestTools;
+
+partial class WindowsTests
+{
+    [UnityTest]
+    [Category("WindowsInput")]
+    public IEnumerator WindowsInput_RemoteDesktopMouseMovements_AreDetected()
+    {
+        var mouse = InputSystem.GetDevice<Mouse>();
```

Comment:
`Mouse.current`

Reply:
You want it moved into the IntegrationTests folder?